addons/vectors/fnc_vectRotate3D.sqf

```diff
@@ -1,26 +1,22 @@
-/*
- * Author: LorenLuke
- * Rotates the first vector around the second, clockwise by angle theta
- *
- * Arguments:
- * 0: Vector <ARRAY>
- * 1: Rotation Axis <ARRAY>
- * 2: Angle (degrees) <NUMBER>
- *
- * Return Value:
- * Transformed Vector <ARRAY>
- *
- * [weaponDir player, [0,0,1], 25] call CBA_fnc_vectorRotate3D
- *
- * Public: No
- */
+ /* ----------------------------------------------------------------------------
+Function: CBA_fnc_vectRotate3D
+Description:
```

Comment:
Missing newline above. Same for Returns and Author below. Might want to add an example as shown here as well:
https://github.com/CBATeam/CBA_A3/blob/master/addons/settings/fnc_init.sqf